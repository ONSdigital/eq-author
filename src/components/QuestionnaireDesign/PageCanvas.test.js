import React from "react";
import { shallow } from "enzyme";
import PageCanvas from "./PageCanvas";

describe("SeamlessInput", () => {
  let component;

  beforeEach(() => {
    component = shallow(<PageCanvas>Children</PageCanvas>);
  });

  it("should render", () => {
    expect(component).toMatchSnapshot();
  });
});
