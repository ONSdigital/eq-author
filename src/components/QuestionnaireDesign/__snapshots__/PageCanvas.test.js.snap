// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SeamlessInput should render 1`] = `
<div
  className="PageCanvas__PageCanvas-o1jumc-0 beftnW"
>
  Children
</div>
`;
